lib/handler.js

```diff
@@ -10,6 +9,8 @@ import {
   fetchRetrievalSuccessRate
 } from './stats-fetchers.js'
 
+import { handlePlatformRoute } from './platform-routes.js'
```

Comment:
```suggestion
import { handlePlatformRoutes } from './platform-routes.js'
```

The number of routes handled by this function should grow over time.